primitives/account/src/account/staking_contract/receipts.rs

```diff
@@ -117,6 +117,12 @@ convert_receipt!(DeleteValidatorReceipt);
 pub struct StakerReceipt {
     // the delegation before this transaction is applied
     pub delegation: Option<Address>,
+    // the active balance before this transaction is applied
+    pub active_balance: Coin,
+    // the inactive release before this transaction is applied
+    pub inactive_release: Option<u32>,
+    // if the staker should reactivate all its stake
+    pub reactivate_all_stake: bool,
```

Comment:
```suggestion
    /// the active balance before this transaction is applied
    pub active_balance: Coin,
    /// the inactive release before this transaction is applied
    pub inactive_release: Option<u32>,
    /// if the staker should reactivate all its stake
    pub reactivate_all_stake: bool,
```
The one before this one was wrong as well.